lib/ansible/cli/galaxy.py

```diff
@@ -251,7 +251,8 @@ def add_install_options(self, parser, parents=None):
                                             "dependencies.".format(galaxy_type))
 
         if galaxy_type == 'collection':
-            install_parser.add_argument('-p', '--collections-path', dest='collections_path', required=True,
+            install_parser.add_argument('-p', '--collections-path', dest='collections_path',
+                                        default=C.COLLECTIONS_PATHS[0],
```

Comment:
The only odd behavior I can create is if I set 'collections_paths = :'  (an empty list)

``` ini
[defaults]
collections_paths = :
```

In that case, the collection ends up getting installed into $PWD/ansible_collections/.
But I suspect that is a general issue with config types of 'pathspec' and not specific
to collection_path.

Reply:
Yeah, this should probably be an error if the config has explicitly removed the paths...